dataprofiler/profilers/profile_builder.py

```diff
@@ -1739,6 +1750,10 @@ def report(self, report_options=None):
                 quantiles = calculate_quantiles(num_quantile_groups, quantiles)
                 report["data_stats"][i]["statistics"]["quantiles"] = quantiles
 
+        if self.options.synthetic_data.is_enabled:
+            report["synthetic_data"] = {
+                "nan_replication": self._nan_replication_metrics
+            }
```

Comment:
would put this in `report["data_stats"][<column_index>]["nan_replication_metrics"]`